src/pages/Punch/ClearPunchWrapper.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState, useEffect } from 'react';
+import { AsyncStatus } from '../../contexts/McAppContext';
+import useCommonHooks from '../../utils/useCommonHooks';
+import {
+    PunchCategory,
+    PunchItem,
+    PunchOrganization,
+    PunchPriority,
+    PunchSort,
+    PunchType,
+} from '../../services/apiTypes';
+import {
+    ClearPunch,
+    PunchEndpoints,
+    UpdatePunchData,
+    useSnackbar,
+    PunchAction,
+} from '@equinor/procosys-webapp-components';
+import Axios from 'axios';
+import usePersonsSearchFacade from '../../utils/usePersonsSearchFacade';
+
+const punchEndpints: PunchEndpoints = {
+    updateCategory: 'SetCategory',
+    updateDescription: 'SetDescription',
+    updateRaisedBy: 'SetRaisedBy',
+    updateClearingBy: 'SetClearingBy',
+    updateActionByPerson: 'SetActionByPerson',
+    updateDueDate: 'setDueDate',
+    updateType: 'SetType',
+    updateSorting: 'SetSorting',
+    updatePriority: 'SetPriority',
+    updateEstimate: 'setEstimate',
+};
+
```

Comment:
Enum?

Reply:
Enums cannot have a type, and since it's important that the punch endpoints in the app are the same as the PunchEndpoints type in the component library I'd rather go with a const to make the fact that they need to match explicit.